### PR TITLE
add reference to Repairing Nodes page

### PR DESCRIPTION
**Pull request by hrischuk-da**

### CODE REVIEW

**Review by danielporterda — APPROVED** [review]

File: docs/2.7.9/docs/deploy-daml/infrastructure-architecture/disaster-recovery.rst [hunk 1]
```diff
@@ -4,9 +4,9 @@
 Disaster Recovery (DR)
 ######################
 
-    Disaster recovery (DR) is the process of maintaining or reestablishing vital infrastructure and systems following a natural or human-induced disaster, such as a storm or battle. It employs policies, tools, and procedures. 
+Disaster recovery (DR) is the process of maintaining or reestablishing vital infrastructure and systems following a natural or human-induced disaster, such as a storm or battle. It employs policies, tools, and procedures. This section presents an overview of DR.  See the section :ref:`Repairing Nodes <repairing-nodes>` for the steps to recover Canton nodes in a DR situation.
```
Comment (carrielaben-da) [inline docs/2.7.9/docs/deploy-daml/infrastructure-architecture/disaster-recovery.rst:7]:
```suggestion
Disaster recovery (DR) is the process of maintaining or reestablishing vital infrastructure and systems following a natural or human-induced disaster, such as a storm or battle. It employs policies, tools, and procedures. This section presents an overview of DR. See :ref:`Repairing Nodes <repairing-nodes>` for the steps to recover Canton nodes in a DR situation. 
```

File: docs/2.8.3/docs/deploy-daml/infrastructure-architecture/disaster-recovery.rst [hunk 2]
```diff
@@ -4,9 +4,9 @@
 Disaster Recovery (DR)
 ######################
 
-    Disaster recovery (DR) is the process of maintaining or reestablishing vital infrastructure and systems following a natural or human-induced disaster, such as a storm or battle. It employs policies, tools, and procedures. 
+Disaster recovery (DR) is the process of maintaining or reestablishing vital infrastructure and systems following a natural or human-induced disaster, such as a storm or battle. It employs policies, tools, and procedures. This section presents an overview of DR.  See the section :ref:`Repairing Nodes <repairing-nodes>` for the steps to recover Canton nodes in a DR situation.
```
Comment (carrielaben-da) [inline docs/2.8.3/docs/deploy-daml/infrastructure-architecture/disaster-recovery.rst:7]:
```suggestion
Disaster recovery (DR) is the process of maintaining or reestablishing vital infrastructure and systems following a natural or human-induced disaster, such as a storm or battle. It employs policies, tools, and procedures. This section presents an overview of DR. See :ref:`Repairing Nodes <repairing-nodes>` for the steps to recover Canton nodes in a DR situation. 
```

File: docs/2.9.0/docs/deploy-daml/infrastructure-architecture/disaster-recovery.rst [hunk 3]
```diff
@@ -4,9 +4,9 @@
 Disaster Recovery (DR)
 ######################
 
-    Disaster recovery (DR) is the process of maintaining or reestablishing vital infrastructure and systems following a natural or human-induced disaster, such as a storm or battle. It employs policies, tools, and procedures. 
+Disaster recovery (DR) is the process of maintaining or reestablishing vital infrastructure and systems following a natural or human-induced disaster, such as a storm or battle. It employs policies, tools, and procedures. This section presents an overview of DR.  See the section :ref:`Repairing Nodes <repairing-nodes>` for the steps to recover Canton nodes in a DR situation.
```
Comment (carrielaben-da) [inline docs/2.9.0/docs/deploy-daml/infrastructure-architecture/disaster-recovery.rst:7]:
```suggestion
Disaster recovery (DR) is the process of maintaining or reestablishing vital infrastructure and systems following a natural or human-induced disaster, such as a storm or battle. It employs policies, tools, and procedures. This section presents an overview of DR. See :ref:`Repairing Nodes <repairing-nodes>` for the steps to recover Canton nodes in a DR situation. 
```

File: docs/3.1/docs/deploy-daml/infrastructure-architecture/disaster-recovery.rst [hunk 4]
```diff
@@ -4,9 +4,9 @@
 Disaster Recovery (DR)
 ######################
 
-    Disaster recovery (DR) is the process of maintaining or reestablishing vital infrastructure and systems following a natural or human-induced disaster, such as a storm or battle. It employs policies, tools, and procedures. 
+Disaster recovery (DR) is the process of maintaining or reestablishing vital infrastructure and systems following a natural or human-induced disaster, such as a storm or battle. It employs policies, tools, and procedures. This section presents an overview of DR.  See the section :ref:`Repairing Nodes <repairing-nodes>` for the steps to recover Canton nodes in a DR situation.
```
Comment (carrielaben-da) [inline docs/3.1/docs/deploy-daml/infrastructure-architecture/disaster-recovery.rst:7]:
```suggestion
Disaster recovery (DR) is the process of maintaining or reestablishing vital infrastructure and systems following a natural or human-induced disaster, such as a storm or battle. It employs policies, tools, and procedures. This section presents an overview of DR. See :ref:`Repairing Nodes <repairing-nodes>` for the steps to recover Canton nodes in a DR situation. 
```

**Review by carrielaben-da — APPROVED** [review]
Small changes only.

**Pull request merged**